quiche/src/lib.rs

```diff
@@ -1394,6 +1394,9 @@ pub struct Connection {
     /// Whether the connection should prevent from reusing destination
     /// Connection IDs when the peer migrates.
     disable_dcid_reuse: bool,
+
+    /// A resusable buffer used by Recovery
+    newly_acked: Vec<recovery::Acked>,
```

Comment:
Why is this not in `Recovery` rather than `Connection`?

Reply:
I explained in the commit message. Because all recovery methods take &mut Recovery, you can't use a field in Recovery